app/Controllers/Http/UserController.js

```diff
@@ -230,12 +298,13 @@ class UserController {
 	async show ({ auth, params, view, response }) {
 		const user = await User.find(Number(params.id));
 		var canEdit = 0;
+		var isAdmin = false;
 		var layoutType = '';
 		// check if admin is viewing their own profile
 		if (auth.user.role === 1) {
 			layoutType = 'layouts/adminLayout';
 			canEdit = 1;
-
+			isAdmin = true;
```

Comment:
isAdmin is redundant. canEdit should be giving the same logic. A perso should be able to see their own bookings so use canEdit as your boolean